src/hyp3_autorift/process.py

```diff
@@ -539,6 +587,10 @@ def main():
 
     product_file, browse_file = process(g1, g2, parameter_file=args.parameter_file, naming_scheme=args.naming_scheme)
 
+    if args.opendata_upload:
+        bucket = 'its-live-data'
+        upload_opendata(product_file, bucket, args.granules[0])
```

Comment:
I believe we want to upload the browse and thumbnail images as well. I suggest writing a single function to determine the bucket prefix to use, then calling upload_file_to_s3 three times, e.g.:
```
prefix = get_opendata_prefix(product_file, args.granules[0])
upload_file_to_s3(product_file, bucket, prefix)
upload_file_to_s3(browse_file, bucket, prefix)
upload_file_to_s3(thumbnail_file, bucket, prefix)
```

Reply:
will implement them later